src/components/controls/LanguageMenu.js

```diff
@@ -0,0 +1,55 @@
+import { createElement } from "react";
+import { ToolbarButton } from "./atoms/ToolbarButton";
+import copy from "../../common/data/copy.json";
+
+export function LanguageMenu({
+  isActive,
+  language: currentLanguage,
+  languages,
+  setLanguage,
+}) {
+  return createElement("div", {
```

Comment:
I'm curious why you are not using JSX here. Is there a specific reason?

Reply:
I thought I had a reason but when I really had think of it I figured it's mostly my personal habits (not used to JSX that has more branching/iteration than markup).

I'll convert it to JSX now, because consistent style across the project has real practical benefits: moving code around is easy and makes useful diffs, grepping for something like `className="..."` works as expected, less surprising in general.

Thanks for bringing it up.